IntegrationTests/3_Components/ShowIdTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
```

Comment:
we also should have unit tests to check at least basic things. Initialisation, and fields values after creating object ;)